Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/MergedLogFileTest.cs

```diff
@@ -71,8 +71,8 @@ public void Test2SmallSources()
 
 				merged.Property(x => x.Count).ShouldEventually().Be(8, TimeSpan.FromSeconds(5),
 																	"Because the merged file should've been finished");
-				merged.Property(x => x.GetValue(LogFileProperties.Size)).ShouldEventually().Be(source0.GetValue(LogFileProperties.Size) + source1.GetValue(LogFileProperties.Size));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.